src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleArray = sourceString.split(';');
+  const styleObject = {};
+
+  styleArray.forEach((style) => {
+    if (style.trim()) {
+      const [key, ...valueParts] = style.split(':');
+
+      if (valueParts.length > 0) {
+        const value = valueParts.join(':').trim();
+
+        styleObject[key.trim()] = value;
```

Comment:
Before assigning the value to `styleObject`, ensure that `key.trim()` is not an empty string. This will prevent adding properties with empty keys.